recipes/scooby/meta.yaml

```diff
@@ -0,0 +1,43 @@
+{% set name = "scooby" %}
+{% set version = "0.4.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: bd3f83dd63566c581523587235fb0e6363424f93a8d71fe21a21e00f60ad52f2
+
+build:
+  number: 0
+  noarch: python
+  script: "{{ PYTHON }} -m pip install . --no-deps -vv"
+
+requirements:
+  host:
+    - python
+    - pip
+  run:
+    - python
+
+test:
+  imports:
+    - scooby
+
+about:
+  home: https://github.com/banesullivan/scooby
+  license: MIT
+  license_family: MIT
+  license_file: LICENSE
+  summary: 'A lightweight toolset to easily report your Python environment's package versions and hardware resources.'
```

Comment:
Yeah you have not quoted the `'` properly. Either you need to use the YAML `|` syntax or you need to have double quotes around the whole string.

Reply:
Ah thanks!